python/ray/tune/tune.py

```diff
@@ -577,16 +576,25 @@ def run(
 
     del remote_run_kwargs
 
-    if os.environ.get("TUNE_RESULT_DIR"):
-        # Deprecate: Raise in 2.6, remove in 2.7
+    # TODO(justinvyu): [Deprecated] Raise in 2.11.
+    if os.environ.get("TUNE_RESULT_DIR") or os.environ.get("RAY_AIR_LOCAL_CACHE_DIR"):
         warnings.warn(
-            "The TUNE_RESULT_DIR environment variable is deprecated and will be "
-            "removed in the future. If you want to set persistent storage to "
-            "a local directory, pass `storage_path` instead. If you are using "
-            "remote storage and want to control the local cache directory, "
-            "set the RAY_AIR_LOCAL_CACHE_DIR environment variable instead.",
-            DeprecationWarning,
+            "The environment variables "
+            "`RAY_AIR_LOCAL_CACHE_DIR` and `TUNE_RESULT_DIR` "
+            "are deprecated and will be removed in the future. "
+            "They are no longer used and will not have any effect. "
+            "You should set the `storage_path` instead. "
+        )
```

Comment:
nit: Split this into two separate if conditions, one per environment variable, so that they user knows exactly what environment variable(s) they have set and need to change.